test/e2e/telemeter_test.go

```diff
@@ -98,17 +97,4 @@ func TestTelemeterRemoteWrite(t *testing.T) {
 			return nil
 		},
 	)
-
-	// Check that the Telemeter server returns no error.
```

Comment:
maybe we can comment out the check and note that it's disabled because flaky?

Reply:
Right maybe add a TODO for the future in order to explain why we don have this test. If a newcomer comes to the repo maybe its difficult to understand why we dont check failed samples.

Reply:
I suppose the question is whether we'll want re-enable it in the future. I would propose that expecting 0 failed samples/s over 5 minutes couples our test too tightly to telemeter and the network conditions in between.

I'm not against bringing a check of this flavor back in the future, just maybe not quite so strict. In that light I'd opt to remove the code.

Reply:
fine by me